public/app/stores/AlertListStore/AlertListStore.ts

```diff
@@ -2,8 +2,8 @@ import { types, getEnv, flow } from 'mobx-state-tree';
 import { AlertRule } from './AlertRule';
 import { setStateFields } from './helpers';
 
-type IAlertRuleType = typeof AlertRule.Type;
-export interface IAlertRule extends IAlertRuleType {}
+type AlertRuleType = typeof AlertRule.Type;
+export interface AlertRule extends AlertRuleType {}
```

Comment:
I dont understand why typescript allows the same type name twice in a file (imported and defined and exported). Same is true in the team store.